proto/celestia/blob/v1/event.proto

```diff
@@ -1,7 +1,7 @@
 syntax = "proto3";
 package celestia.blob.v1;
 
-option go_package = "github.com/celestiaorg/celestia-app/v2/x/blob/types";
+option go_package = "github.com/celestiaorg/celestia-app/x/blob/types";
```

Comment:
[question] when do we bump the version number `v1` in `package celestia.blob.v1`?

> You can add an optional package specifier to a .proto file to prevent name clashes between protocol message types.

> In Go, the package directive is ignored, and the generated .pb.go file is in the package named after the corresponding go_proto_library Bazel rule. For open source projects, you must provide either a go_package option or set the Bazel -M flag.

Ref: https://protobuf.dev/programming-guides/proto3/#packages

It seems like the `package` statement is optional so I wonder if we should consider removing them to avoid confusion especially if they're not doing anything. Can definitely be a separate issue to investigate.